src/charts/bar.js

```diff
@@ -1292,6 +1300,22 @@ define(function(require) {
             return this;
         };
 
+        /**
+         * Gets or Sets the locale which d3-format uses.
+         * (Default is en-US)
+         * @param  {string} _x                   Desired locale
```

Comment:
You can do [_x = 'es-US'] to mark the default.

Could you add a link in the comment to point to the possible values?
Also, if we accept a locale object instead of the string, we should explain its values or add another link for referencing the format of it.

Reply:
Fixed